lib/ui/router/branch_container.dart

```diff
@@ -17,8 +17,8 @@ class BranchContainer extends StatelessWidget {
     return Stack(
       children: [
         ...children.mapIndexed(
-          (index, navigator) => Opacity(
-            // TODO: [https://github.com/FlutterKaigi/conference-app-2023/issues/145]
+          (index, navigator) => AnimatedOpacity(
+            duration: const Duration(milliseconds: 400),
```

Comment:
```suggestion
            duration: const Duration(
              milliseconds: 400,
            ),
```

Reply:
thanks for review.
fixed at https://github.com/FlutterKaigi/conference-app-2023/pull/159/commits/f858de3d1f5999d12bc3c2afea27129d9fa912c3